qr_generator/lib/repositories/seed_repository.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:math';
+
+import 'package:flutter/foundation.dart';
+import 'package:qr_generator/model/seed_model.dart';
+import 'package:qr_generator/utils/api_client.dart';
+
+class SeedRepository {
+  final ApiClient netUtil;
```

Comment:
**question:** are you familiar with dependency injection?

Reply:
I'm familiar with it and have used it occasionally, but I don't use it frequently but I'm looking forward to learning more about it and starting using it more.